automation/project/Dependencies.scala

```diff
@@ -5,6 +5,13 @@ object Dependencies {
   val akkaV = "2.4.17"
   val akkaHttpV = "10.0.5"
 
+  val workbenchModelV  = "0.7-3fc9a3b"
+  val workbenchModel: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-model" % workbenchModelV
```

Comment:
What exactly is workbench-model needed for? With this pulled in, we have to be careful to not use code that clients don't have (such as serialization) that would make the tests stay green even if we make a breaking change to the request/response format.

Reply:
Probably because `GoogleIamDAO` (in workbench-google) uses the `WorkbenchIdentity` case classes in workbench-model. Good point about not using workbench-libs JSON serializers in test code -- I don't think we'd be using the JSON serializers in this case, but we could make sure we _don't_ import `WorkbenchIdentityJsonSupport` in any tests.